CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 0.63.0      2024-11-21
```

Comment:
The version in the code was updated to 0.62.1, but this is 0.63.0?

If we want to do a patch release I'd assume it would branch off of the previous release instead of releasing the main branch.

Reply:
There's probably no harm in going ahead and releasing 0.63.0. We're already running this version of the collector in production.